inclusive-design/wecount.inclusivedesign.ca
# Article bottom padding (resolves #300)

## Description
Adds padding to bottom of article of main content and adjust the spacing of the pagination component between the last post item and footer.


## Steps to test

1. Go to News page in iPad display.
2. Scroll down to the last post item.
3. Observe space between last post item and footer.
4. Go to Views page examine the adjusted spacing of the pagination component between the last post item and footer.

**Expected behavior:** 
In iPad display there should be space between the last News/Views item and footer. The spacing of the pagination component between the last post item and footer should also be adjusted.

## Additional information
N/A


## Related issues
closes #300
